### ISSUE TITLE
Error: Cannot load report metadata

### ISSUE THREAD

**Issue by k0d3r1s** [NoiseByNorthwest/php-spx#264]
Hi, I can't seem to open any report anymore, worked fine like a week ago.

spx returns 404 for, example, `https://.../?SPX_UI_URI=/data/reports/metadata/spx-full-20240909_135310-62eef133cd59-31-1825596768` although `spx-full-20240909_135310-62eef133cd59-31-1825596768` do exist in /tmp/spx as `.json` and `.txt.gz`. Opening control panel works ok, reports are generated and listed. `https://.../?SPX_UI_URI=/data/metrics` works and loads

what am i missing?

```
SPX Support | enabled
-- | --
SPX Version | 0.4.16

[SPX](https://.../_profiler/phpinfo#module_spx)
SPX Support 	enabled
SPX Version 	0.4.16
Directive	Local Value	Master Value
spx.data_dir	/tmp/spx	/tmp/spx
spx.debug	0	0
spx.http_enabled	1	1
spx.http_ip_var	REMOTE_ADDR	REMOTE_ADDR
spx.http_ip_whitelist	*	*
spx.http_key	vairogs	vairogs
spx.http_profiling_auto_start	no value	no value
spx.http_profiling_builtins	no value	no value
spx.http_profiling_depth	no value	no value
spx.http_profiling_enabled	no value	no value
spx.http_profiling_metrics	no value	no value
spx.http_profiling_sampling_period	no value	no value
spx.http_trusted_proxies	*	*
spx.http_ui_assets_dir	/usr/share/misc/php-spx/assets/web-ui	/usr/share/misc/php-spx/assets/web-ui
```

I am on php 8.4 currently but I have the same issue on php 8.3 as well

**Comment by NoiseByNorthwest** [comment]
This is a regression (not covered by a tag) caused by #255

**Comment by k0d3r1s** [comment]
@NoiseByNorthwest could be similar but no, same issue in 0.4.16 tag which is before #255 merge

**Comment by NoiseByNorthwest** [comment]
I've just tested v0.4.16, it is OK. Are you sure you have properly switched to v0.4.16 ?

**Comment by NoiseByNorthwest** [comment]
@k0d3r1s fixed since the merge of #265

**Comment by k0d3r1s** [comment]
@NoiseByNorthwest thanks, checked, works. will check in 0.4.16 when have time
